editor/code_editor.cpp

```diff
@@ -1769,6 +1769,11 @@ void CodeTextEditor::_zoom_popup_id_pressed(int p_idx) {
 	_zoom_to(zoom_button->get_popup()->get_item_metadata(p_idx));
 }
 
+void CodeTextEditor::_indentation_button_pressed() {
+	set_indent_using_spaces(!text_editor->is_indent_using_spaces());
+	text_editor->convert_indent();
```

Comment:
Is it common in IDEs for this to actually convert the whole file indentation?

At least in Kate which I'm familiar with, the similarly placed indentation menu does not convert the file, it just affects the indentation for future code that you write.